source/guide/3.1/plugin-docker.md

```diff
@@ -189,3 +191,153 @@ Here are the operations that this plugin currently supports:
 ### Delete task
 
 * Deletes the container and its runtime_properties.
+
+
+# Nodecellar Example
```

Comment:
Maybe change to "Complete Example" or "Example Application", not sure everyone know what Nodecellar is